pkg/syncer/cnsoperator/manager/init.go

```diff
@@ -195,6 +195,27 @@ func InitCnsOperator(ctx context.Context, clusterFlavor cnstypes.CnsClusterFlavo
 					return err
 				}
 				log.Infof("%q CRD is created successfully", cnsoperatorv1alpha1.CnsUnregisterVolumePlural)
+
+				// Clean up routine to cleanup successful CnsUnregisterVolume instances.
+				log.Info("Starting go routine to cleanup successful CnsUnregisterVolume instances.")
```

Comment:
Should we add any checks to make sure only successful CnsUnregisterVolume CRs are deleted. PTAL.

Reply:
It's already there in `cleanUpCnsUnregisterVolumeInstances` method. See line 95.